crm_location/__openerp__.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Python source code encoding : https://www.python.org/dev/peps/pep-0263/
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright :
+#        (c) 2015 Antiun Ingenieria, SL (Madrid, Spain, http://www.antiun.com)
+#                 Endika Iglesias <endikaig@antiun.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    # Addon information
+    'name': 'CRM location',
+    'category': 'Customer Relationship Management',
+    'version': '1.0',
+    'depends': [
+        'crm',
+        'base_location',
+    ],
+    'external_dependencies': {},
+    # Views templates, pages, menus, options and snippets
+    'data': [
+        'views/crm_lead_view.xml',
+    ],
+    # Qweb templates
+    'qweb': [
+    ],
+    # Your information
+    'author': 'Antiun Ingeniería S.L.',
+    'website': 'http://www.antiun.com',
```

Comment:
Add `Odoo Community Association (OCA)` as co-author for visibility in app store.